modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -64,7 +72,9 @@ endif::[]
 
 2. Egress router for OVN-Kubernetes supports only redirect mode.
 
-3. IPv6 is supported only on bare metal, vSphere, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+3. IPv4-primary dual-stack networking on bare-metal, {vmw-full}, {ibm-power-name}, {ibm-z-name}, and {rh-openstack} platforms.
+
+4. IPv6 single-stack networking on a bare-metal platform.
 
-4. IPv6 single stack is not supported on {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+5. IPv6-primary dual-stack networking for a cluster running on a bare-metal, a {vmw-full}, or an {rh-openstack} platform.
```

Comment:
Should this match the construction of `3.`?
```suggestion
5. IPv6-primary dual-stack networking on a bare-metal, {vmw-full}, or {rh-openstack} platforms.
```

Reply:
:heavy_check_mark: